backend/src/main/java/com/redhat/cloud/notifications/routers/handlers/endpoint/EndpointResource.java

```diff
@@ -681,15 +699,6 @@ public void testEndpoint(@Context SecurityContext sec, @IntegrationId @RestPath
         this.endpointTestService.testEndpoint(internalEndpointTestRequest);
     }
 
-    private static void checkSystemEndpoint(EndpointType endpointType) {
```

Comment:
Can we safely deploy this before the frontend is done transitioning to the new drawer and email integration management? I assume it's safe, but I want to double check.